bindings/csharp/net40/Program.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
```

Comment:
should we add a proper C# test suite before merging to main?

Reply:
we could do it in later stages. there are a couple of things we can do but let's first have the base line into the main.